src/bridge/cockpitpackages.c

```diff
@@ -1166,6 +1158,10 @@ cockpit_packages_new (void)
 
   packages = g_new0 (CockpitPackages, 1);
 
+  packages->locale = cockpit_language_from_locale (g_getenv ("LANG"));
+  if (!packages->locale)
+    packages->locale = g_strdup ("c");
```

Comment:
With `setlocale` this should then trigger a warning, as it defaults to "C" by itself (or maybe C.UTF-8 in some future).